FirebaseAnalyticsSwift.podspec

```diff
@@ -0,0 +1,30 @@
+Pod::Spec.new do |s|
+  s.name                    = 'FirebaseAnalyticsSwift'
+  s.version                 = '7.9.0-beta'
+  s.summary                 = 'Swift Extensions for Firebase Analytics'
+
+  s.description      = <<-DESC
+Firebase Analytics is a free, out-of-the-box analytics solution that inspires actionable insights based on app usage and user engagement.
+                       DESC
+
+  s.homepage                = 'https://firebase.google.com/features/analytics/'
+  s.license                 = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors                 = 'Google, Inc.'
+
+  s.source                  = {
+    :git => 'https://github.com/Firebase/firebase-ios-sdk.git',
+    :tag => 'CocoaPods-' + s.version.to_s
+  }
+
+  s.swift_version           = '5.0'
+  s.ios.deployment_target   = '13.0'
+
+  s.cocoapods_version       = '>= 1.10.0'
+  s.prefix_header_file      = false
+
```

Comment:
Missing platforms specification. Presumably, it's iOS only?

Reply:
I thought I could get away with `spec.ios.deployment_target` alone, but I'll try to add the platform specification as well.

Reply:
No luck, still running into the same issue. I can reproduce locally, except Xcode seems to know what's going on (autocomplete works for both `import FirebaseAnalytics` and Analytics specific symbols).